testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveStorageFormats.java

```diff
@@ -433,6 +452,26 @@ public void testInsertAndSelectWithNullFormat(StorageFormat storageFormat)
         onTrino().executeQuery(format("DROP TABLE %s", tableName));
     }
 
+    @Test(dataProvider = "storageFormatsWithZeroByteFile", groups = {STORAGE_FORMATS_DETAILED, HMS_ONLY})
+    public void testSelectFromZeroByteFile(StorageFormat storageFormat)
+    {
+        String tableName = format(
+                "test_storage_format_%s_zero_byte_file",
+                storageFormat.getName());
+        hdfsClient.saveFile(format("%s/%s/zero_byte", warehouseDirectory, tableName), nullInputStream());
+        onTrino().executeQuery("DROP TABLE IF EXISTS " + tableName);
+        onTrino().executeQuery(format(
+                "CREATE TABLE %s" +
+                        " (name varchar) " +
+                        "WITH ( " +
+                        "   format = '%s'" +
+                        ")",
+                tableName,
+                storageFormat.getName()));
+        assertThat(onTrino().executeQuery("SELECT * FROM " + tableName)).hasNoRows();
+        onTrino().executeQuery("DROP TABLE IF EXISTS " + tableName);
```

Comment:
`IF EXISTS` is redundant here, remove

Reply:
not done.

Reply:
Oh - done actally - but in wrong commit. Same for changes regarding comments above